test/testcases/history.exp

```diff
@@ -5,7 +5,7 @@
 .EXT tar -tj < history-test.tar.bz2 2> /dev/null | head -1
 .EXT tar -xj < history-test.tar.bz2
 Source: history-test.tar.bz2
-Created on: Fri 14 Dec 19:08:38 UTC 2012
+Created on: Sat 15 Dec 03:08:38 CST 2012
```

Comment:
This is a strange change, why has it switched from UTC to local time zone? Hrm..

Reply:
I just use “regression.sh prepare” to generate it

Reply:
Might have to set the locale or TZ in the container